### PR TITLE
[CRX] Fix native messaging links

### PR DESCRIPTION
**Pull request by AmySteam**
✅ **Scope**
- Adds links to new Native Messaging extension docs
- Converts numbered links to text links

❌ **Does not**
- Fix any outdated information.

### CODE REVIEW

File: site/en/docs/extensions/mv3/messaging/index.md [hunk 1]
```diff
@@ -313,51 +313,27 @@ chrome.tabs.sendMessage(tab.id, {greeting: "hello"}, function(response) {
...
+[runtime-on-disconnect]: /docs/extensions/reference/runtime#property-Port-onDisconnect
+[runtime-on-msg-ext]: /docs/extensions/reference/runtime#event-onMessageExternal
+[runtime-on-msg]: /docs/extensions/reference/runtime#event-onMessage
+[runtime-port-disconnect]: /docs/extensions/reference/runtime#property-Port-disconnect
+[runtime-port]: /docs/extensions/reference/runtime#type-Port
+[runtime-post-msg]: /docs/extensions/reference/runtime#property-Port-postMessage
+[runtime-send-msg]: /docs/extensions/reference/runtime#method-sendMessage
+[section-external]: #external
+[section-long]: #connect
+[section-one]: #simple
```
Comment (sebastianbenz) [inline site/en/docs/extensions/mv3/messaging/index.md:333]:
```suggestion
[section-one-time]: #simple
```

**Review by sebastianbenz — APPROVED** [review]
Looks good - one broken link.

File: site/en/docs/extensions/mv3/messaging/index.md [hunk 2]
```diff
@@ -141,25 +141,25 @@ chrome.runtime.onConnect.addListener(function(port) {
...
 
 You may want to find out when a connection is closed, for example if you are maintaining separate
-state for each open port. For this listen to the [runtime.Port.onDisconnect][21] event. This
+state for each open port. For this listen to the [runtime.Port.onDisconnect][runtime-on-disconnect] event. This
 event is fired when there are no valid ports at the other side of the channel. This happens in the
 following situations:
 
-- There are no listeners for [runtime.onConnect][22] at the other end.
+- There are no listeners for [runtime.onConnect][runtime-on-connect] at the other end.
 - The tab containing the port is unloaded (e.g. if the tab is navigated).
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/messaging/index.md:159]:
```suggestion
- The tab containing the port is unloaded (for example, if the tab is navigated).
```

File: site/en/docs/extensions/mv3/messaging/index.md [hunk 3]
```diff
@@ -262,24 +262,24 @@ chrome.runtime.onMessageExternal.addListener(
...
-[native messaging host][35]. To learn more about this feature, see [Native messaging][36].
+Extensions [can exchange messages][native-client] with native applications that are registered as a
+[native messaging host][native-host]. To learn more about this feature, see [Native messaging][native-messaging].
 
 ## Security considerations {: #security-considerations }
 
 ### Content scripts are less trustworthy {: #content-scripts-are-less-trustworthy }
 
-[Content scripts are less trustworthy][37] than the extension background page (e.g., a malicious web
+[Content scripts are less trustworthy][security-cs] than the extension service worker (e.g., a malicious web
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/messaging/index.md:272]:
```suggestion
[Content scripts are less trustworthy][security-cs] than the extension service worker (for example, a malicious web
```

**Review by rachelandrew — APPROVED** [review]
I just removed a couple of e.g.'s that you didn't put in, but while I was in there, I figured I'd fix them. The style guide says not to use e.g. or i.e.

**Pull request merged**